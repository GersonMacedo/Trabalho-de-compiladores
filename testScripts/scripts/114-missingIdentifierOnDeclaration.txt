! -s SYNTACTIC
! should complain about missing identifier token
program p ;
    var : boolean ;
begin
end .