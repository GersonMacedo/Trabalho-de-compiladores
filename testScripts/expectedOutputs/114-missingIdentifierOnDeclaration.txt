
Performing up to syntactic analysis at testScripts/scripts/114-missingIdentifierOnDeclaration.txt
Parser::Expected '<identifier>' but ':' was found at line 4 column 9
